### PR TITLE
Homework/task 10

### PR DESCRIPTION
**Pull request by AlexanderPhilippov**

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (mmarkelov) [inline cypress/fixtures/example.json:5]: I think this is unnecessary file

File: cypress/integration/e2e/main.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/// <reference types="cypress" />
+
+describe('E2E Tests', () => {
+    it('Search, open details and close it after assert', () => {
+        // Open site
+        cy.visit('http://127.0.0.1:3000')
```
Comment (mmarkelov) [inline cypress/integration/e2e/main.js:6]: `http://127.0.0.1:3000` can be a constant to reuse it in test

File: cypress/support/commands.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (mmarkelov) [inline cypress/support/commands.js:1]: Also this seems like unnecessary file

File: cypress/fixtures/example.json (hunk 1 above)
Comment (AlexanderPhilippov) [reply cypress/fixtures/example.json]: Removed

File: cypress/integration/e2e/main.js (hunk 2 above)
Comment (AlexanderPhilippov) [reply cypress/integration/e2e/main.js]: Done

File: cypress/support/commands.js (hunk 3 above)
Comment (AlexanderPhilippov) [reply cypress/support/commands.js]: Removed

**Pull request merged**